.changes/unreleased/tinymce-TINY-10901-2024-05-14.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Iframes in dialogs was not rendering rounded borders correctly.
```

Comment:
```suggestion
body: Iframes in dialogs were not rendering rounded borders correctly.
```

Reply:
Time to install a grammar checker in nvim. 😄